docs-source/spring/content/development/_index.md

```diff
@@ -0,0 +1,38 @@
+This section provides information about how to develop and deploy Spring Boot applications
+with the Oracle Backend for Spring Boot and Microservices. 
+
+Spring Boot applications can be developed the normal way, with no special requirements, and
+be deployed into Oracle Backend for Spring Boot and Microservices.  However, if you do opt-in to the platform
```

Comment:
The sentence "However, if you ..." doesn't make sense. Should "provides" be changed to "provided"?